core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -225,22 +225,26 @@ private void sendImpression(@Nonnull ProjectConfig projectConfig,
                                 @Nonnull Experiment experiment,
                                 @Nonnull String userId,
                                 @Nonnull Map<String, ?> filteredAttributes,
-                                @Nonnull Variation variation) {
-        if (!experiment.isRunning()) {
-            logger.info("Experiment has \"Launched\" status so not dispatching event during activation.");
-            return;
-        }
+                                @Nonnull Variation variation,
+                                @Nonnull String flagKey,
```

Comment:
it should be featureFlagKey or featureKey

Reply:
we are calling it flagKey and flagType in all other sdks. So should I change it in java-sdk?